File: php/native/HelloWorldKomencoNative/Operation.php
<?php

namespace Automatski\Komenco;

class Operation {
    private $gate;
    private $qubits;
    private $params;

    public function __construct(string $gate, array $qubits, array $params) {
        $this->gate = $gate;
        $this->qubits = $qubits;
        $this->params = $params;
    }

    public function getGate(): string {
        return $this->gate;
    }

    public function getQubits(): array {
        return $this->qubits;
    }

    public function getParams(): array {
        return $this->params;
    }
}

?>

File: c#/native/HelloWorldKomencoNative/Operation.cs
using System.Collections.Generic;

namespace Automatski.Komenco
{
    public class Operation
    {
        public string Gate { get; }
        public List<int> Qubits { get; }
        public List<double> Params { get; }

        public Operation(string gate, List<int> qubits, List<double> parameters)
        {
            this.Gate = gate;
            this.Qubits = qubits;
            this.Params = parameters;
        }

        public string GetGate()
        {
            return Gate;
        }

        public List<int> GetQubits()
        {
            return Qubits;
        }

        public List<double> GetParams()
        {
            return Params;
        }
    }
}